### PR TITLE
0.15.2

### PR DESCRIPTION
**Pull request by minouHub**
# Minor fall 2024 release: Bugs and compatibility with Mac OS

## What's Changed
* Exomolapi broadening bug by @HajimeKawahara in https://github.com/radis/radis/pull/691
* Bug fix for downloading the HDO exomol database by @HajimeKawahara in https://github.com/radis/radis/pull/693
* Compatibility with numpy 2.0 by @minouHub in https://github.com/radis/radis/pull/688
* Mac OS: testing and compatibility  by @minouHub in https://github.com/radis/radis/pull/697

**Full Changelog**: https://github.com/radis/radis/compare/0.15.1...0.15.2

### CODE REVIEW

File: radis/api/dbmanager.py [hunk 1]
```diff
@@ -41,7 +41,7 @@
 import pandas as pd
 from dateutil.parser import parse as parse_date
 from joblib import Parallel, delayed
-from numpy import DataSource
+from numpy.lib.npyio import DataSource
```
Comment (erwanp) [inline radis/api/dbmanager.py:44]: Seems it's only after numpy>=2.0.0 ?  https://numpy.org/doc/1.26/reference/generated/numpy.DataSource.html#numpy-datasource

To be sure, you can also do a try/except

```suggestion
try:
    from numpy.lib.npyio import DataSource
except ImportError:  # numpy <2.0.0?
    from numpy import DataSource
```

File: radis/tools/database.py [hunk 2]
```diff
@@ -993,20 +993,17 @@ def query(df, conditions="", **kwconditions):
...
                 query.append(f"{k} != {k}")
             else:
                 #                    query.append('{0} == {1}'.format(k,v))
-                query.append("{0} == {1}".format(k, v.__repr__()))
-                # ... for some reason {1}.format() would remove some digit
-                # ... to floats in Python2. Calling .__repr__() keeps
-                # ... the correct format, and has no other consequences as far
-                # ... as I can tell
-
+                # query.append("{0} == {1}".format(k, v.__repr__()))
```
Comment (erwanp) [inline radis/tools/database.py:1004]:
```suggestion
```

**Review by erwanp — DISMISSED** [review]

File: radis/api/dbmanager.py (hunk 1 above)
Comment (minouHub) [reply radis/api/dbmanager.py:44]: yes indeed, it's only after numpy 2.0

**Review by HajimeKawahara — APPROVED** [review]
Thanks! the current version is working well with ExoJAX!

**Comment by minouHub** [comment]
closed and re-open to launch Travis tests (free credits renewed)

**Pull request merged**